chart/compass/charts/director/templates/deployment.yaml

```diff
@@ -89,6 +89,20 @@ spec:
             - name: APP_PAIRING_ADAPTER_SRC
               value: /pairing-adapters/config.json
             {{ end }}
+          livenessProbe:
+              httpGet:
+                  port: {{.Values.deployment.args.containerPort }}
+                  path: "/healthz"
+              initialDelaySeconds: {{ .Values.deployment.liveness.initialDelaySeconds }}
+              timeoutSeconds: {{ .Values.deployment.liveness.timeoutSeconds }}
+              periodSeconds: {{.Values.deployment.liveness.periodSeconds }}
+          readinessProbe:
+              httpGet:
+                  port: {{.Values.deployment.args.containerPort }}
+                  path: "/ready"
+              initialDelaySeconds: {{ .Values.deployment.readiness.initialDelaySeconds }}
```

Comment:
Where can I find this value?

Reply:
good catch, I remove this configuration option